getting_started_moa/creating-first-moa-cluster.adoc

```diff
@@ -0,0 +1,46 @@
+:context: creating-first-moa-cluster
+[id="creating-first-moa-cluster"]
+= Getting started with {product-title}
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+This guide walks through setting up your first {product-title} cluster using `moactl`, the {product-title} command line utility.
+
+If you have already installed the required prerequisites, here are the commands you need to create a cluster.
+
+[source, bash]
+----
+## Configures your AWS account and ensures everything is setup correctly
+$ moactl init
+
+## Starts the cluster creation process (~30-40minutes)
+$ moactl create cluster --name <my-cluster-name>
+
+## Watch your logs as your cluster creates
+$ moactl logs cluster <my-cluster-name> --watch
+
+## Connect your IDP to your cluster
+$ moactl create idp --cluster <my-cluster-name>  --interactive
+
+## Promotes a user from your IDP to admin level
+$ moactl create user --cluster <my-cluster-name> --dedicated-admins <admin-username>        
+
+## Checks if your install is ready (look for State: Ready),
+## and provides your Console URL to login to the web console.
+$ moactl describe cluster <my-cluster-name>
+----
+
+For additional help, consult the rest of this guide for steps on setting up and running an {product-title} cluster on your AWS account.
+
+By the end of this guide you will have an Amazon Red Hat OpenShift cluster running in your AWS account.
```

Comment:
Will there be a special token for this product name?

Reply:
I combined these two sentences into one and added the product-title token. We're using {product-title}, flagged in the distro file as Amazon Red Hat OpenShift.
(separate PR - https://github.com/openshift/openshift-docs/pull/23896/files#diff-bd14e794ee15cd5fc66c6e1b13376fbbR140)